### DISCUSSION TITLE
19.10.2 (revision 9245) on Linux Lite 5.6(based on Ubuntu 20.04), normalize audio filter not working

### DISCUSSION THREAD

**Discussion by funk192** [smplayer-dev/smplayer#338] [Q&A]
SMPlayer audio filter normalize doesn't seem to do anything at all. I'm somewhat a newbie to Linux and I cannot seem to get the audio filters to work. I've installed SMPlayer through the Synaptic package manager, but I did try flatpak before and that also didn't seem to work. Is there anything I am missing?

**Comment by smplayer-dev** [comment]
Maybe the normalize audio filter is wrong. Do the other audio filters work for you?

**Reply by funk192** [reply]
Seems like the other filters do actually do something, the Extra Stereo filter for some reason makes it louder, karaoke and headphone also seem to do something.

Is this type of normalization filter meant to boost quiet sounds?
